pkg/master/client_ca_hook_test.go

```diff
@@ -189,18 +189,20 @@ func TestWriteClientCAs(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		client := fake.NewSimpleClientset(test.preexistingObjs...)
-		test.hook.tryToWriteClientCAs(client.Core())
+		t.Run(test.name, func(t *testing.T) {
+			client := fake.NewSimpleClientset(test.preexistingObjs...)
+			test.hook.tryToWriteClientCAs(client.Core())
 
-		actualConfigMaps, updated := getFinalConfiMaps(client)
-		if !reflect.DeepEqual(test.expectedConfigMaps, actualConfigMaps) {
-			t.Errorf("%s: %v", test.name, diff.ObjectReflectDiff(test.expectedConfigMaps, actualConfigMaps))
-			continue
-		}
-		if test.expectUpdate != updated {
-			t.Errorf("%s: expected %v, got %v", test.name, test.expectUpdate, updated)
-			continue
-		}
+			actualConfigMaps, updated := getFinalConfiMaps(client)
+			if !reflect.DeepEqual(test.expectedConfigMaps, actualConfigMaps) {
+				t.Fatalf("%s: %v", test.name, diff.ObjectReflectDiff(test.expectedConfigMaps, actualConfigMaps))
+				return
```

Comment:
Please send a follow-up to omit the return statement, it's redundant now.

@hzxuzhonghu please notice things like this before LGTMing, thanks :)

Reply:
Oh wait, this didn't actually merge? :) Please fix :)

Reply:
sorry for mistake

Reply:
Sorry, it's my negligence. I will fix it right away. thank all.

Reply:
Done